apps/taxes/src/modules/taxjar/order-created/taxjar-order-created-payload-transformer.test.ts

```diff
@@ -1,22 +1,21 @@
 import { describe, expect, it } from "vitest";
-import { taxJarMockFactory } from "../taxjar-mock-factory";
+import { TaxJarOrderCreatedMockGenerator } from "./taxjar-order-created-mock-generator";
 import {
   TaxJarOrderCreatedPayloadTransformer,
   sumPayloadLines,
 } from "./taxjar-order-created-payload-transformer";
-import { taxMockFactory } from "../../taxes/tax-mock-factory";
-
-const MOCKED_PAYLOAD = {
-  order: taxMockFactory.createOrderCreatedMock(),
-  channelConfig: taxJarMockFactory.createMockChannelConfig(),
-};
 
 describe("TaxJarOrderCreatedPayloadTransformer", () => {
   it("returns the correct order amount", () => {
+    const mockGenerator = new TaxJarOrderCreatedMockGenerator();
```

Comment:
you can lift instantion out the test block, so we have better structure
- arranging happens at first, globally 
- test block contains only code that is important for the scenario

Reply:
We can do it whenever possible, but when we are testing different scenarios that are defined in the mocked generator class, we must initialize it with the scenario (`new TaxJarOrderCreatedMockGenerator("scenarioA")`, `new TaxJarOrderCreatedMockGenerator("scenarioB")`). In that case, the initialization would be part of the test.

I will lift it here though

Reply:
Ah ok, I thought scenario is part of method. Thats fine